unittest/CMakeLists.txt

```diff
@@ -21,21 +21,23 @@ SET(${PROJECT_NAME}_CPP_TESTS
   test_diff_actions
   test_impulses
   test_multiple_impulses
+  test_costs
```

Comment:
Better to run test_friction_cone first since the friction-cone cost depends on it.